servatrice/src/smtp/qxtsmtp.cpp

```diff
@@ -23,39 +23,44 @@
  **
  ****************************************************************************/
 
-
 /*!
  * \class QxtSmtp
  * \inmodule QxtNetwork
  * \brief The QxtSmtp class implements the SMTP protocol for sending email
  */
 
-
-
 #include "qxtsmtp.h"
-#include "qxtsmtp_p.h"
+
 #include "qxthmac.h"
-#include <QStringList>
-#include <QTcpSocket>
+#include "qxtsmtp_p.h"
+
 #include <QNetworkInterface>
 #include <QSslSocket>
+#include <QStringList>
+#include <QTcpSocket>
 
 QxtSmtpPrivate::QxtSmtpPrivate() : QObject(0)
 {
     // empty ctor
 }
 
-QxtSmtp::QxtSmtp(QObject* parent) : QObject(parent)
+QxtSmtp::QxtSmtp(QObject *parent) : QObject(parent)
 {
     QXT_INIT_PRIVATE(QxtSmtp);
     qxt_d().state = QxtSmtpPrivate::Disconnected;
     qxt_d().nextID = 0;
     qxt_d().socket = new QSslSocket(this);
     QObject::connect(socket(), SIGNAL(encrypted()), this, SIGNAL(encrypted()));
-    //QObject::connect(socket(), SIGNAL(encrypted()), &qxt_d(), SLOT(ehlo()));
+    // QObject::connect(socket(), SIGNAL(encrypted()), &qxt_d(), SLOT(ehlo()));
     QObject::connect(socket(), SIGNAL(connected()), this, SIGNAL(connected()));
     QObject::connect(socket(), SIGNAL(disconnected()), this, SIGNAL(disconnected()));
-    QObject::connect(socket(), SIGNAL(error(QAbstractSocket::SocketError)), &qxt_d(), SLOT(socketError(QAbstractSocket::SocketError)));
+#if (QT_VERSION >= QT_VERSION_CHECK(6, 0, 0))
```

Comment:
the formatting changes here make this hard to figure out, is this the only change in the entire file?
is it worth reformatting the entire file that is borrowed from the old libqxt anyway?

Reply:
The file auto-reformatted with clang. I ran through it manually and I only made one change to the file (What oyu see here). Sorry for the overlaps.